Tests/LibTimeZone/TestTimeZone.cpp

```diff
@@ -126,4 +131,11 @@ TEST_CASE(get_time_zone_offset)
     EXPECT(!TimeZone::get_time_zone_offset("I don't exist"sv, {}).has_value());
 }
 
+#else
+
+TEST_CASE(built_with_time_zone_data)
```

Comment:
Typo?

Reply:
I don't see it :thinking:

Reply:
`built_with_time_zone_data`, `EXPECT(!TimeZone::built_with_time_zone_data());`

Reply:
oh, I'm stupid, I assumed the test checks that it's built with timezone data, but it's checking the function named `built_with_time_zone_data` :sweat_smile:

Reply:
It's gone now anyway :^)